cmd/ttn-lw-stack/commands/config.go

```diff
@@ -43,7 +44,8 @@ type Config struct {
 	JS               joinserver.Config        `name:"js"`
 	Console          console.Config           `name:"console"`
 	GCS              struct {
-		BasicStation cups.ServerConfig `name:"basic-station"`
+		BasicStation     cups.ServerConfig `name:"basic-station"`
+		TheThingsGateway ttg_cups.Config   `name:"the-things-gateway"`
```

Comment:
Note: This configuration will instead be added to `gatewayconfigurationserver.Config` once the #435 is merged.